README.md

```diff
@@ -1,34 +1,89 @@
-# RpaEmAssemblyWebapp
+# @hmcts/dg-docassembly-webcomponent 
+[![Coverage Status](https://coveralls.io/repos/github/hmcts/dg-docassembly-webcomponent/badge.svg?branch=master)](https://coveralls.io/github/hmcts/dg-docassembly-webcomponent?branch=upload-npm-in-pipeline)
+[![Build Status](https://travis-ci.com/hmcts/dg-docassembly-webcomponent.svg?branch=master)](https://travis-ci.com/hmcts/dg-docassembly-webcomponent)
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 7.1.4.
+The docassembly webcomponent is an angular library that provides components to generate a document based on a predefined template customised with user input.
+The library is made up of 3 main components:
+- assembly-viewer, which is the parent component that acts as a wrapper that wires the other components together
+- form-viwer, which retrieves and displays the input form that allows the user to cutomise the generated document
+- document-viewer, which enables the user to preview the generated document
 
-## Development server
+### Integration options
+- default setup, wiring between internal components already done
+- service-driven setup, more control to the service to define the styling of the component, but also requires the service
+  to do the wiring between the form-viewer and the document-viewer (more details below) 
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+### Building document assembly web component
+- npm run package
+- distributable will be created under dist/dg-docassembly-webcomponent
 
-## Run docker dependencies:
+### Add as a dependency in your angular app
+- add @hmcts/dg-docassembly-webcomponent as a dependency in package.json
+- import AssemblyModule and declare it in your NgModule imports.
 
-```
-az acr login --name hmcts --subscription 1c4f0704-a29e-403d-b719-b90c34ef14c9
-docker-compose -f docker-compose-dependencies.yml up
-```
+  For example:
+  ```
+  import { AssemblyModule } from 'dg-docassembly-webcomponent';
 
-## Code scaffolding
+  @NgModule({
+    imports: [
+      ...,
+      AssemblyModule,
+    ]
+  })
+  ```
+- import assets to your angular.json
+  ```
+    {
+        "glob": "**/*",
+        "input": "projects/dg-docassembly-webcomponent/node_modules/@hmcts/document-viewer-webcomponent/assets",
+        "output": "/assets"
+    }
+  ```
+- and styles
+  ```
+  "styles": [
+    "projects/dg-docassembly-webcomponent/node_modules/@hmcts/document-viewer-webcomponent/assets/aui-styles.scss",
```

Comment:
for assets as well, we discussed this but as a reminded, im not sure the path is right as their structure isnt gonna be projects so it might be node_modules/@hmcts/......